libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/GetConversationMembers.cs

```diff
@@ -95,5 +93,21 @@ protected override string OnComputeId()
         {
             return $"{GetType().Name}[{this.Property?.ToString() ?? string.Empty}]";
         }
+
+        private static async Task<IEnumerable<ChannelAccount>> GetMembersAsync(IConnectorClient connectorClient, string conversationId, CancellationToken cancellationToken)
```

Comment:
>IConnectorClient connectorClient, string conversationId [](http://example.com/codeflow?start=79&length=55)

Can we ITurnContext here instead of connectorClient and conversationId?

Reply:
Sure, I've pushed a change to simplify this area of the code. Thanks for the suggestion.